src/cplscheme/CompositionalCouplingScheme.cpp

```diff
@@ -194,6 +172,17 @@ double CompositionalCouplingScheme::getTimeWindowSize() const
   return timeWindowSize;
 }
 
+bool CompositionalCouplingScheme::getIsCoarseModelOptimizationActive() const
+{
+  // TODO @BU: what's the desired behaviour for a CompositionalCouplingScheme?
+  return false;
+}
+
+void CompositionalCouplingScheme::activateCoarseModelOptimization()
+{
+  // TODO @BU: what's the desired behaviour for a CompositionalCouplingScheme?
```

Comment:
An assertion would be good.

Reply:
Done. c29a6e33a88d34b02b605c96d121950ce5a3e0e8